relay-server/src/metrics_extraction/transactions.rs

```diff
@@ -318,9 +335,13 @@ mod tests {
             panic!(); // Duration must be set
         }
 
+        let user_metric = &metrics[6];
+        assert_eq!(user_metric.name, "sentry.transactions.user");
+        assert!(matches!(user_metric.value, MetricValue::Set(_)));
+
         assert_eq!(metrics[1].tags["measurement_rating"], "meh");
 
-        for metric in metrics {
+        for metric in &metrics[0..6] {
             assert!(matches!(metric.value, MetricValue::Distribution(_)));
```

Comment:
i would keep all assertions for all metrics except for this one

Reply:
That makes sense.